### ISSUE TITLE
Create dbt Documentation and Testing 10x faster with ChatGPT

### ISSUE THREAD

**Issue by pbritosa-sage** [fishtown-analytics/docs.getdbt.com#3367]
### Contact Details

pedro@hellosage.com

### I have read the dbt Developer Blog contribution guidelines.

- [X] I have read the dbt Developer Blog contribution guidelines.

### Which of these best describes you?

- [X] I am a dbt Community member or partner contributing to the Developer Blog
- [ ] I work for dbt Labs and am creating this issue for a community or marketing approved piece.

### What is the topic of your post?

Whether you are creating your pipelines into dbt for the first time or just adding a new model once in a while, good documentation and testing should always be a priority for you and your team. But good documentation and testing is time-consuming and a hassle to write. This blog post provides a set of simple guidelines to leverage ChatGPT in order to automate most of the work in generating the YAML files with model descriptions and tests.

### Link to an initial outline.

I have shared my experience with using ChatGPT to generate docs and tests with my assigned dbt Solution Architect Luis Leon (luis.leon@dbtlabs.com) and he suggested that I write this blog post.
Happy to change the form and content in any way you see fit.

This doc has several page but it's mostly due to very large images and tables of results. Actual content is < 1000 words and easily "trimmable".
https://docs.google.com/document/d/1gdrLOyCxuTs7JvGVCzrOV9_JcsLrSp0FHivVTR22rTQ/edit?usp=sharing

**Comment by mirnawong1** [comment]
hey @pbritosa-sage , thanks for opening this issue up and what a great blog post idea! I've looped in @KiraFuruichi as well to this as it relates to the devhub blog and someone will get in touch with more info 😄

**Comment by pbritosa-sage** [comment]
Hey @mirnawong1 thank you so much! @KiraFuruichi let me know if I can help with anything, looking forward to having this in the blog!

**Comment by pbritosa-sage** [comment]
Hi any update on this?

**Comment by mirnawong1** [comment]
hey @pbritosa-sage , apologies for the delay here! due to the recent RIF and staff changes, we've had to readjust some things but i'm going to work with a teammate to help get this published for you. i will come back to you soon 🙏

**Comment by jasnonaz** [comment]
Hi @pbritosa-sage - thanks for the post. You can find a preview of this live in the PR above. Can you let us know in that PR if [the post](https://deploy-preview-3748--docs-getdbt-com.netlify.app/blog/create-dbt-documentation-10x-faster-with-ChatGPT) and [your bio](https://deploy-preview-3748--docs-getdbt-com.netlify.app/author/pedro_brito_de_sa) look good and if so we'll get this ready to go live 💪

**Comment by pbritosa-sage** [comment]
Hi @jasnonaz , is it ok to ask you to replace all code snippets and pictures with the new ones I added in the original Docs file? I noticed that the resolution of the images was awful so hopefully now they will be a bit better, but my main concern was that I was actually showing some field names that I'd rather not show for security reasons. I had to replicate everything again on ChatGPT so the content is slightly different but the message is the same.

Additionally, I would just like to replace this sentence:

> 'This meant 30+ of facts and dimensions into a Staging Layer,'
with
'This meant 30+ tables of internal facts and dimensions + external data into a Staging Layer'

Everything else looks great.

Sorry for the last minute changes and duplication of your work, thank you for understanding!
